src/cls/rbd/cls_rbd.cc

```diff
@@ -4132,6 +4134,25 @@ int mirror_image_status_remove_down(cls_method_context_t hctx, bufferlist *in,
   return 0;
 }
 
+/**
+ * Initialize the header with basic metadata.
+ * Everything is stored as key/value pairs as omaps in the header object.
+ *
+ * Output:
+ * @return 0 on success, negative error code on failure
+ */
+int create_cg(cls_method_context_t hctx, bufferlist *in, bufferlist *out)
+{
+  bufferlist snap_seqbl;
+  uint64_t snap_seq = 0;
+  ::encode(snap_seq, snap_seqbl);
+  int r = cls_cxx_map_set_val(hctx, CG_SNAP_SEQ, &snap_seqbl);
```

Comment:
The idea is the same as for images - keep the latest id we assigned to its snapshot. So far this code is only a place holder. Do you want me to remove it until we introduce snapshots?